packages/js/client-config-builder/package.json

```diff
@@ -45,5 +47,41 @@
   "gitHead": "7346adaf5adb7e6bbb70d9247583e995650d390a",
   "publishConfig": {
     "access": "public"
+  },
+  "doc-snippets": {
+    "extract": {
+      "include": "./**/*.{ts,tsx,json,yaml,txt,md,graphql,cue}",
+      "ignore": [
+        "./**/node_modules/**",
+        "./**/.polywrap/**",
+        "./**/__tests__/**"
+      ],
+      "dir": "./src"
+    },
+    "inject": {
+      "dir": "./readme",
+      "include": "./README.md"
+    },
+    "startTokens": [
+      {
+        "pattern": "$start: ",
+        "inline": false
+      },
+      {
+        "pattern": "/* $: {SNIPPET_NAME} */",
```

Comment:
Why do we have `{SNIPPET_NAME}` here, but not in the above start token?

Reply:
Oh nvm, I think I figured it out, it's because this one is "inline" so it can't know where to extract the snippet name from like it can in the pattern above. Is that correct?

Reply:
right, that's correct. The `{SNIPPET_NAME}` helps define a regex pattern.